package.json

```diff
@@ -29,17 +29,79 @@
 		"Other"
 	],
 	"activationEvents": [
-		"onCommand:ocm-vscode-extension.ocmNewProject"
+		"onCommand:ocm-vscode-extension.ocmNewProject",
+		"onCommand:ocm-vscode-extension.verifyTools",
+		"onCommand:ocm-vscode-extension.getClusteradmVersion",
+		"onCommand:ocm-vscode-extension.buildLocalClusters",
+		"onView:ocm.explorer"
 	],
 	"main": "./out/src/extension.js",
 	"contributes": {
 		"commands": [
 			{
 				"command": "ocm-vscode-extension.ocmNewProject",
-				"title": "create an OCM application-manager project",
+				"title": "Create an application-manager project",
 				"category": "OCM"
+			},
+			{
+				"command": "ocm-vscode-extension.verifyTools",
+				"title": "Verify required tools existence",
+				"category": "OCM"
+			},
+			{
+				"command": "ocm-vscode-extension.getClusteradmVersion",
+				"title": "Get clusteradm version",
+				"category": "OCM"
+			},
+			{
+				"command": "ocm-vscode-extension.buildLocalClusters",
+				"title": "Build local clusters",
+				"category": "OCM"
+			}
+		],
+		"viewsContainers": {
+			"activitybar": [
+				{
+					"id": "ocmView",
+					"title": "OCM",
+					"icon": "images/icon-black.svg"
+				}
+			]
+		},
+		"views": {
+			"ocmView": [
+				{
+					"id": "forDevelopers",
+					"name": "Developers Tools"
+				},
+				{
+					"id": "managedClustersExplorer",
+					"name": "Managed Clusters Explorer"
+				},
+				{
+					"id": "hubClustersExplorer",
+					"name": "Hub Clusters Explorer"
+				},
+				{
+					"id": "deployedApplication",
+					"name": "Deployed Application"
+				}
+			]
+		},
+		"viewsWelcome": [
+			{
+				"view": "forDevelopers",
+				"contents": "[Check Clusteradm Version](command:ocm-vscode-extension.getClusteradmVersion)\n [Verify Tools Existence](command:ocm-vscode-extension.verifyTools) \n  [Start Local Clusters](command:ocm-vscode-extension.buildLocalClusters)"
 			}
 		],
+		"menus": {
+			"view/item/context": [
+				{
+					"command": "ocm-vscode-extension.ocmNewProject",
+					"when": "view == ocmView && viewItem == createProject"
```

Comment:
I don't think this works,
we don't have a _createProject_ view item named in our _ocmView_ view.